tsconfig.json

```diff
@@ -15,5 +15,6 @@
     "listEmittedFiles": true,
     "resolveJsonModule": true
   },
-  "include": ["src/**/*"]
+  "include": ["src/**/*"],
+  "exclude": ["src/tests/*", "package.json"]
```

Comment:
So these won't get added into the `dist` dir when building thru TypeScript compiler